Gopkg.toml

```diff
@@ -65,5 +65,33 @@
   name = "github.com/giantswarm/microkit"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```

Comment:
This is to be reverted once https://github.com/giantswarm/operatorkit/pull/319 gets merged.